scripts/sandboxGenerator.sh

```diff
@@ -0,0 +1,182 @@
+#! /bin/bash
+
+# Script that executes the following operations :
+# 1- Execute the repackage goal to generate the MAC springBoot jar in the state of the repository
+# 2- Executes a maven goal that builds a customized (no real time) Sandbox springboot jar for a given version of Activepivot
+# 3- Run both apps
+# 4- Generate the memory statistics file for the Sandbox
+# 5- Load the generated statisitics in the MAC app
+# 6- Run queries on the MAC app
+# 7- Kill both apps
+
+# REQUIREMENTS :
+# Java 11+
+# Maven 3+
+# ActiveViam artifacts access
+# Valid ActivePivot License
+# jq
+
+# INPUT : 
+# - $1 String : AP version
+
+# OUTPUT : 
+# - File : Query & server logs
```

Comment:
maybe a help section could be nice unless this script will be ported to CI

Reply:
I don't really see the point of adding an interactive --help section for a code that is to be run by the ci (and eventually locally but in that case the commented source code is accessible )
That would be useful if this scipt was bound to be used in a cli, but I don't think it is.